mobile/src/models/TabContent.ts

```diff
@@ -0,0 +1,4 @@
+export type TabContent = {
+    video?: string;
+    body: string;
+};
```

Comment:
Can we move this to Book.ts? I think it makes sense to keep them together - we're never going to reuse this type for anything other than books.